lib/bootloader_setup.pm

```diff
@@ -334,7 +334,7 @@ sub uefi_bootmenu_params {
     # assume bios+grub+anim already waited in start.sh
     # in grub2 it's tricky to set the screen resolution
     #send_key_until_needlematch('grub2-enter-edit-mode', 'e', 5, 0.5);
-    (is_jeos) ? send_key_until_needlematch('grub2-enter-edit-mode', 'e', 5, 0.5)
+    (is_jeos) ? send_key_until_needlematch('grub2-enter-edit-mode', 'e', 5, 0.05)
```

Comment:
this is reducing the timeout as well in `check_screen($tag, $timeout)`. Why?

Reply:
Just a test if it changes anything, but apparently no change, the failure rate is still high.